First commit and smoll changes.

Trippeh opened the pull request:
- https://github.com/zerratar/Ravenfall-Overlay/commit/33bb9e9ac707620cc3827ef7e5eeb08b192319c8 First commit, smoll changes:
- - Removed old resources.
- - Added new skills.
- - Added new 'non-formatted' coins value.
- - Added myself to devs and myself as debug user.
- - Fixed a comment not being commented out properly.

- https://github.com/zerratar/Ravenfall-Overlay/commit/716d8b8796ed4000615fd6112d3757142fc8c829 Finish adding new islands.

- https://github.com/zerratar/Ravenfall-Overlay/commit/2ca6701c9594d97ef27fa448bbd60ada37ff6449 REALY Finish adding the new islands.

- https://github.com/zerratar/Ravenfall-Overlay/commit/68f9997e35959a2df99189fb7b619123524b2180 Relocated var skillNames:
- - Moved to utilities.js
- - Didn't belong in states.js

- https://github.com/zerratar/Ravenfall-Overlay/commit/719b945547a58c068025559132ce842d79f1b1fe Refactored the API variables:
- - 1 unused and 1 redundant
- - Fix UI for Islands View:
- - - Less redundant text
- - - Formatting per island is fixed because of this too.

- https://github.com/zerratar/Ravenfall-Overlay/commit/1ecae51765238ac51e85ffb6bb4249412b29e606 Further simplified and fixed the newly refactored API variables in states.js.

Trippeh committed:
First commit, smoll changes:
- Removed old resources.
- Added new skills.
- Added new 'non-formatted' coins value.
- Added myself to devs and myself as debug user.
- Fixed a comment not being commented out properly.

Trippeh committed:
Finish adding new islands.

Trippeh committed:
REALY Finish adding the new islands.

Trippeh committed:
Relocated var skillNames:
- Moved to utilities.js
- Didn't belong in states.js

Trippeh committed:
Refactored the API variables:
- 1 unused and 1 redundant

Fix UI for Islands View:
- Less redundant text
- Formatting per island is fixed because of this too.

Trippeh committed:
Simplified/Fixed Reference URLs in states.js

zerratar commented:
I can only see two direct issues

https://github.com/zerratar/Ravenfall-Overlay/commit/1ecae51765238ac51e85ffb6bb4249412b29e606#r128608652
and
https://github.com/zerratar/Ravenfall-Overlay/commit/68f9997e35959a2df99189fb7b619123524b2180#r128608588

I can merge if fixed

Trippeh committed:
proposed fixes for previous commits:
- No longer creating a new var for ravenfallHost.
- Changed herbalism (derp) into alchemy.

Trippeh commented:
Done

zerratar reviewed (approved):
All good, I dont see any direct issues.
